home/tests/test_add_answer.py

```diff
@@ -1,28 +1,5 @@
-from home.models import Profile, Answer
+from home.models import Answer
 import pytest
-from django.contrib.auth import authenticate
-from test_question_tag_model_functions import question_test_data
-
-@pytest.fixture
-def profile():
-    profile = Profile.create(username="test_user", password="testtest", email="test@test.com")
-    return profile
-
-
-@pytest.mark.django_db
-def test_default_params_profile(profile):
-    assert profile.user.id >= 0  # check if user saved successfully and get an id
-    assert profile.gender == 'U'  # default gender is U
-    assert profile.is_blocked is False
-
-
-@pytest.fixture
-def authenticated_user(client, request):
-    username = 'Lior'
-    password = 'LiorLior'
-    client.login(username=username, password=password)
-    user = authenticate(request, username=username, password=password)
-    return user
```

Comment:
Why didn't you rebased it with the previous commit? I just spent time reviewing a commit that's apparently is not relevant because u made changes in it...

Reply:
I did fixup instead of squash, sorry for that my mistake.. 
fixed it now